src/python/grpcio_tests/tests/unit/_rpc_test.py

```diff
@@ -225,6 +225,7 @@ def testSuccessfulUnaryRequestBlockingUnaryResponseWithCall(self):
 
         self.assertEqual(expected_response, response)
         self.assertIs(grpc.StatusCode.OK, call.code())
+        self.assertEqual(call.debug_error_string(), "")
```

Comment:
Throughout gRPC Python we use `self.assertEqual(<expected>, <obtained from system under test>)` order.

Reply:
Done